lib/tasks/update_https.rake

```diff
@@ -22,44 +22,36 @@ namespace :usasearch do
     end
   end
 
-  def info(msg)
```

Comment:
@noremmie & @nickmarden - 

TIL, the hard way: https://kevinjalbert.com/defined_methods-in-rake-tasks-you-re-gonna-have-a-bad-time/

I ran into an issue where, deep in the bowels of some resque code I was working on, `@redis.info` was being called, and `@redis.info` was defined as...this rake task method:
```
> show-method @redis.info
From: /Users/Moth/src/GSA/search-gov/lib/tasks/update_https.rake @ line 25:
Owner: Object
Visibility: private
Number of lines: 5

def info(msg)
  STDOUT.puts(msg) unless Rails.env.test?
end
```

I didn't realize that methods within rake tasks aren't namespaced(!). At any rate, I moved the `puts` silencing to the test via a new `quiet_puts` helper method.  I'm leaving the other methods in this task for now, as their names are less likely to collide with others. That said, I have learned my lesson and will no longer put methods in my rake tasks.

Note: the Redis team is aware of this pitfall, and will avoid it in future versions:
```
Passing 'info' command to redis as is; administrative commands cannot be effectively namespaced and should be called on the redis connection directly; passthrough has been deprecated and will be removed in redis-namespace 2.0 (at /Users/Moth/.rvm/gems/ruby-2.3.5@searchgov-rails42/gems/resque-1.27.4/lib/resque/data_store.rb:59:in `method_missing')
```

Reply:
lol, ouch.